README.md

```diff
@@ -213,51 +231,75 @@ using a combination of `bwa` and `samtools` to output a BAM file instead of a SA
 
-- If the appropriate test data doesn't exist for your module then it MUST be added to [`tests/data/`](tests/data/).
+- Any outputs produced by the test workflow MUST be included in the [pytest-workflow](https://pytest-workflow.readthedocs.io/en/stable) for that tool e.g. [`tests/software/fastqc/test.yml`](tests/software/fastqc/test.yml).  `md5sum` checks are the preferable choice of test to determine file changes, however, this may not be possible for all outputs generated by some tools e.g. if they include time stamps or command-related headers. Please do your best to avoid just checking for the file being present e.g. it may still be possible to check that the file contains the appropriate text snippets.
 
-- A filter for the module must be created in [`.github/filters.yml`](.github/filters.yml). Please include any paths specific for that tool or upstream of that tool (For example bowtie build is upstream of bowtie align).
+- A filter for the module must be created in [`.github/filters.yml`](.github/filters.yml). If the test workflow you have created invokes more than one tool please include any paths specific for those tool's too e.g. `bowtie build` is upstream of `bowtie align` and they have both been chained together to test the latter.
 
 #### Running Tests Locally
 
-0. Have either `docker`, `singularity` or `conda` installed
-1. See [pytest-workflow installation](https://pytest-workflow.readthedocs.io/en/stable/#installation) for directions to install
-2. Run
+1. Install [`nextflow`](https://nf-co.re/usage/installation)
 
-``` bash
-PROFILE=docker pytest --tag new_module --symlink --wt 2 --kwdof
-PROFILE=conda pytest --tag new_module --symlink --wt 2 --kwdof
-TMPDIR=~ PROFILE=singularity pytest --tag new_module --symlink --wt 2 --kwdof
-alias nf-test="PROFILE=docker pytest --tag new_module --symlink --wt 2 --kwdof"
-```
+2. Install any of [`Docker`](https://docs.docker.com/engine/installation/), [`Singularity`](https://www.sylabs.io/guides/3.0/user-guide/) or [`Conda`](https://conda.io/miniconda.html)
+
+3. Install [`pytest-workflow`](https://pytest-workflow.readthedocs.io/en/stable/#installation)
+
+4. Start running your own tests!
+
+    - Typical command with Docker:
+
+        ```console
+        cd /path/to/git/clone/of/nf-core/modules/
+        PROFILE=docker pytest --tag tests/software/bowtie/build --symlink --wt 2 --keep-workflow-wd
+        ```
+
+    - Typical command with Singularity:
+
+        ```console
+        cd /path/to/git/clone/of/nf-core/modules/
+        TMPDIR=~ PROFILE=singularity pytest --tag tests/software/bowtie/build --symlink --wt 2 --keep-workflow-wd
+        ```
+
+    - Typical command with Conda:
+
+        ```console
+        cd /path/to/git/clone/of/nf-core/modules/
+        PROFILE=conda pytest --tag tests/software/bowtie/build --symlink --wt 2 --keep-workflow-wd
```

Comment:
These tags aren't right. They may work but I think they're not the best direction to put people on. We should be able to get by with just `--tag bowtie_build`

Reply:
Just confirmed these tags don't work

Reply:
Ok. Let me try.

Reply:
`--tag bowtie_build` doesn't work for me because that one is specifically broken as we know! Sorry, it was late and I was speed writing docs! However, I do know why I put the path down. It's because you can also run the tests without a `--tag` but with the full path to the folder containing the `test.yml`

e.g.
```console
TMPDIR=~ PROFILE=singularity pytest tests/software/bowtie/build/ --symlink --wt 2 --keep-workflow-wd
```

This test still works even with the issues we were having with getting `bowtie build`.